lib/src/services/coin_service/app_coin_service.dart

```diff
@@ -0,0 +1,43 @@
+import 'dart:convert';
+
+import 'package:http/http.dart' as http;
+import '/../src/config/app_config.dart';
+import '/../src/config/constants.dart';
+
+import '/../src/models/coins.dart';
+import '/../src/services/coin_service/coin_service.dart';
+
+class AppAlbumService extends CoinService {
+  AppAlbumService(http.Client client) : super(client);
+
+  @override
+  Future<List<Coins>> fecthCoins() async {
+    Map<String, String> qParams = {
+      AppConfig.instance.getValue(AppConstants.currency): 'usd',
+      AppConfig.instance.getValue(AppConstants.order): 'market_cap_desc',
+      AppConfig.instance.getValue(AppConstants.perPage): '20',
+      AppConfig.instance.getValue(AppConstants.page): '1',
+      AppConfig.instance.getValue(AppConstants.sparkline): 'false',
+    };
```

Comment:
Should move default parameters into constants or config file

Reply:
I will change it.